zazu/repo/commands.py

```diff
@@ -81,12 +89,93 @@ def clone(ctx, repository_url, nohooks, nosubmodules):
 
 
 @repo.command()
+@click.option('-i', 'interactive', is_flag=True, help='starts interactive repo init')
 @click.pass_context
-def init(ctx):
-    """Initialize repo directory structure"""
-    raise NotImplementedError
-
+def init(ctx, interactive):
+    """Initialize repo directory structure
+    TODo:
+    -if not git repo offer to make one
+    -if git repo, assume user wants current repo to use zazu
+    """
+    
+    def _get_plugin_list(plugin_subclass):
+        """helper function to pull lists of plugins"""
+        plugins = straight.plugin.load('zazu.plugins', subclasses=plugin_subclass)
+        known_types = {p.type().lower(): p.from_config for p in plugins}
+        return known_types.keys()
+
+    def _build_inquiry(name, message, choices=[], checkbox=False):
+        """helper function to build an inquiry"""
+        if not checkbox:
+            inquiry = [
+                inquirer.List(str(name),
+                             message = str(message),
+                             choices = trackers,
+                             ),
+            ]
+
+        else:
+            inquiry = [
+                inquirer.Checkbox(str(name),
+                             message = str(message),
+                             choices = choices,
+                             ),
+            ]
+        
+        return inquirer.prompt(inquiry)
+
+    def _zazu_yaml(issue_tracker={}, stylers={}):
+        """builds zazu.yml file"""
+        zazu_yaml_obj = {}
+        zazu_yaml_obj['issueTracker'] = issue_tracker
+        if stylers:
+            zazu_yaml_obj['style'] = stylers['style']
+        yaml.dump(zazu_yaml_obj, file('zazu.yaml','w'), default_flow_style=False)
+
+    def _getZazuYaml():
+        """gets zazu's yaml"""
+        import pdb
+        pdb.set_trace()
+        zazu_yaml = requests.get('https://raw.githubusercontent.com/stopthatcow/zazu/develop/zazu.yaml')
```

Comment:
Might need to for ci as there are many possibilities when designing a build plan, I figure for this i will scrap the use of zazu's yaml and build the interactive designer to support issue trackers and stylers. Once we have ci as a plugin we use empty templates.